tests/testfont.c

```diff
@@ -103,6 +103,8 @@ static void verifyFont (GpFont *font, GpFontFamily *originalFamily, INT expected
 	status = GdipGetFontStyle(font, &style);
 	assertEqualInt (status, Ok);
 	assertEqualInt (style, expectedStyle);
+	
+	GdipDeleteFontFamily (family);
```

Comment:
I looked at it with gdb and it was crashing in the `FcFini()` call in `GdiplusShutdown()`, presumably because this wasn't cleaned up properly.